apps/enc.c

```diff
@@ -560,7 +560,7 @@ int enc_main(int argc, char **argv)
     }
  end:
     ERR_print_errors(bio_err);
-    OPENSSL_free(strbuf);
+    OPENSSL_clear_free(strbuf, SIZE);
```

Comment:
There is call to OPENSSL_cleanse around line 627. Can you actually confirm that it's not sufficient?